doc/style-guide.md

```diff
@@ -44,3 +45,52 @@ quaility.
     character limit.
 
 **[top](#table-of-contents)**
+
+## Expressions
+
+  - Expressions in return statements should be surrounded by parenthesis for clarity.
```

Comment:
Yeah I'm mostly being pedantic here. :) I just wanted a word that captures both `return a` and `return !a` (the latter being what I think of when I hear "one operand"), but can't think of something beyond "Complex expressions in return statements..." which isn't great either.